projects/addon-editor/components/toolbar-tools/highlight-color/highlight-color.component.ts

```diff
@@ -17,10 +20,10 @@ import {distinctUntilChanged, map} from 'rxjs/operators';
 export class TuiHighlightColorComponent {
     @Input()
     @tuiDefaultProp()
-    colors: ReadonlyMap<string, string> = defaultEditorColors;
+    colors: ReadonlyMap<string, string> = this.defaultOptions.colors;
 
     readonly backgroundColor$ = this.editor.stateChange$.pipe(
-        map(() => this.editor.getBackgroundColor() || EDITOR_BLANK_COLOR),
```

Comment:
Where does this blank color come from? Do you know why it's `rgb(51, 51, 51)`?

Reply:
Unfortunately, I dont know :(